### PR TITLE
Feature/job description screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Code UI and logic get data job and show to view job

## Task
Link of task: 

1. https://trello.com/c/kOEmxPWj
2. https://trello.com/c/bIH0WZ9f

## Screenshot
Screenshot_1697021784

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/29dc28b1-c5a1-4b60-95a9-470b1b4cb4a3

### CODE REVIEW

File: lib/src/presentations/add_job/screen/add_job_view.dart [hunk 1]
```diff
@@ -154,15 +157,87 @@ class AddJobView extends StatelessWidget {
...
     );
   }
 
+  Widget _buildJobRequirements() {
+    return BlocBuilder<AddJobCubit, AddJobState>(
+      buildWhen: (previous, current) =>
+          previous.requirements != current.requirements,
+      builder: (context, state) {
+        return _buildJobInfo(
+          title: "Requirements",
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_job/screen/add_job_view.dart]: using .arb

File: lib/src/presentations/add_job/screen/add_job_view.dart [hunk 2]
```diff
@@ -154,15 +157,87 @@ class AddJobView extends StatelessWidget {
...
+                      index: index,
+                    );
+                  },
+                  separatorBuilder: (context, index) =>
+                      const SizedBox(height: 15),
+                  itemCount: state.requirements.length,
+                ),
+          onTap: () {
+            AddJobCoordinator.showAddJobDescription(
+              title: " Job Requirement",
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_job/screen/add_job_view.dart]: the same above

File: lib/src/presentations/add_job/screen/add_job_view.dart [hunk 3]
```diff
@@ -154,15 +157,87 @@ class AddJobView extends StatelessWidget {
...
+  }) {
+    return Row(
+      children: [
+        Expanded(
+          child: Text(requirement, style: AppStyles.normalTextMulledWine),
+        ),
+        GestureDetector(
+          onTap: () {
+            AddJobCoordinator.showAddJobDescription(
+              title: " Job Requirement",
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_job/screen/add_job_view.dart]: using .arb

File: lib/src/presentations/add_job/screen/add_job_view.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_job/screen/add_job_view.dart]: I fixed it

File: lib/src/presentations/add_job/screen/add_job_view.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_job/screen/add_job_view.dart]: I fixed it

File: lib/src/presentations/add_job/screen/add_job_view.dart (hunk 3 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_job/screen/add_job_view.dart]: I fixed it

**Pull request merged**